main-cookbooks/teracy-dev/recipes/apt.rb

```diff
@@ -120,6 +120,15 @@
 end
 
 
+execute 'apt-get-update-periodic' do
```

Comment:
this is already included on apt::default recipe and we already add apt to chef run_list, seem duplicated.